core/src/main/java/com/exadel/kaliada/core/models/ByLineModel.java

```diff
@@ -60,6 +63,6 @@ private Image getImage(){
 
     @PostConstruct
     private void init(){
-        System.out.println("Hello");
+        logger.info("init ByLineModel");
```

Comment:
That's OK but not needed in production code)
+ (cosmetic) We always add spaces between `() {`